app/services/analytics_events.rb

```diff
@@ -5947,11 +5959,18 @@ def piv_cac_recommended_visited
   # Tracks when user's piv cac setup
   # @param [Boolean] in_account_creation_flow Whether user is going through account creation
   # @param [Integer] enabled_mfa_methods_count Number of enabled MFA methods on the account
-  def piv_cac_setup_visited(in_account_creation_flow:, enabled_mfa_methods_count: nil, **extra)
+  # @param [Integer] attempts number of MFA setup attempts
+  def piv_cac_setup_visited(
+      in_account_creation_flow:,
+      enabled_mfa_methods_count: nil,
+      attempts: nil,
+      **extra
+    )
```

Comment:
Nit: Indentation is off

```suggestion
    in_account_creation_flow:,
    enabled_mfa_methods_count: nil,
    attempts: nil,
      **extra
  )
```

Reply:
De-dented that